### PR TITLE
added name

### PR DESCRIPTION
**Pull request by davidaceves**

### CODE REVIEW

**Comment by mmargo10** [comment]
I like your commit messages. They give a very nice picture of your work flow.

File: assignments/prototype-refactor.js [hunk 1]
```diff
@@ -7,3 +8,218 @@ Prototype Refactor
...
+    let newLine = '\n'
+  
+    console.log(mage.createdAt); // Today's date
+    console.log(archer.dimensions); // { length: 1, width: 2, height: 4 }
+    console.log(swordsman.healthPoints); // 15
+    console.log(mage.name); // Bruce
+    console.log(swordsman.team); // The Round Table
+    console.log(mage.weapons); // Staff of Shamalama
+    console.log(archer.language); // Elvish
+    console.log(archer.greet()); // Lilith offers a greeting in Elvish.
```
Comment (mmargo10) [inline assignments/prototype-refactor.js]: Really good job refactoring!

File: assignments/lambda-classes.js [hunk 2]
```diff
@@ -1 +1,181 @@
 // CODE here for your Lambda Classes
+
+class Person {
```
Comment (mmargo10) [inline assignments/lambda-classes.js]: All of your classes look great! Your code is super clean and well-organized. You met all of the requirements for MVP plus stretch. Well done!

**Comment by davidaceves** [comment]
Thanks for all the reviews. I'm going to redo my closures for JS II.

David Aceves

**Pull request merged**